examples/docs_snippets/docs_snippets/concepts/ops_jobs_graphs/dep_dsl.py

```diff
@@ -28,7 +28,7 @@ def construct_graph_with_yaml(yaml_file, op_defs) -> GraphDefinition:
 
     deps = {}
 
-    for op_yaml_data in yaml_data["ops"]:
+    for op_yaml_data in yaml_data["ops"]:  # type: ignore[index]
```

Comment:
I have a PR up specifically for this fix that I think might be a better approach - having `load_yaml_from_path` return a more specific type rather than returning `object`:

https://github.com/dagster-io/dagster/pull/10144